src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Main.cs

```diff
@@ -174,6 +177,101 @@ public void Init(PluginInitContext context)
             _context = context;
         }
 
+        public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
+        {
+            if (selectedResult?.ContextData is not VSCodeWorkspace workspace)
+            {
+                return new List<ContextMenuResult>();
+            }
+
+            string realPath = SystemPath.RealPath(workspace.RelativePath);
+
+            return new List<ContextMenuResult>
+            {
+                CreateContextMenuResult(
+                    title: $"{Resources.CopyPath} (Ctrl+C)",
+                    glyph: "\xE8C8", // Copy
+                    acceleratorKey: Key.C,
+                    acceleratorModifiers: ModifierKeys.Control,
+                    action: () => CopyToClipboard(realPath)
+                ),
+                CreateContextMenuResult(
+                    title: $"{Resources.OpenInExplorer} (Ctrl+Shift+F)",
+                    glyph: "\xEC50", // File Explorer
+                    acceleratorKey: Key.F,
+                    acceleratorModifiers: ModifierKeys.Control | ModifierKeys.Shift,
+                    action: () => OpenInExplorer(realPath)
+                )
+                CreateContextMenuResult(
+                    title: $"{Resources.OpenInConsole} (Ctrl+Shift+C)",
+                    glyph: "\xE756", // Command Prompt
+                    acceleratorKey: Key.C,
+                    acceleratorModifiers: ModifierKeys.Control | ModifierKeys.Shift,
+                    action: () => OpenInConsole(realPath)
+                ),
+            };
+        }
+
+        private ContextMenuResult CreateContextMenuResult(string title, string glyph, Key acceleratorKey, ModifierKeys acceleratorModifiers, Func<bool> action)
+        {
+            return new ContextMenuResult
+            {
+                PluginName = Name,
+                Title = title,
+                Glyph = glyph,
+                FontFamily = "Segoe Fluent Icons,Segoe MDL2 Assets",
+                AcceleratorKey = acceleratorKey,
+                AcceleratorModifiers = acceleratorModifiers,
+                Action = context => action()
+            };
+        }
+
+        private bool CopyToClipboard(string path)
+        {
+            try
+            {
+                Clipboard.SetText(path);
+                return true;
+            }
+            catch (Exception)
+            {
+                ShowErrorMessage("Can't copy to clipboard");
+                return false;
```

Comment:
Please add logging using `Wox.Plugin.Logger.Log.Exception()`.

(Same on the other places.)

Reply:
@htcfreek

Hey! 😊

I'm using Visual Studio Code, and I need to generate the `Resources.Designer.cs` file from a `.resx` file. Can you guide me on how to do that in VSCode?

Thanks in advance! 🙏

Reply:
@htcfreek
The code is ready for review. Please share any feedback or suggestions. Thanks!